psi4/driver/driver_cbs.py

```diff
@@ -916,58 +1077,208 @@ def return_energy_components():
 
     # Integrate CFOUR methods
     VARH.update(cfour_psivar_list())
+
+    # Integrate density functionals
```

Comment:
Please find some other word than "integrate" - it makes me think of numerical integration grids, which I don't believe is what you mean.